components/automate-ui/src/app/entities/desktop/desktop.effects.ts

```diff
@@ -181,6 +181,11 @@ export class DesktopEffects {
       ofType(DesktopActionTypes.UPDATE_DESKTOPS_FILTER_CURRENT_PAGE),
       mergeMap(() => [ new GetDesktops() ]));
 
+  @Effect()
+  updateDesktopFilterPageSize$ = this.actions$.pipe(
+    ofType(DesktopActionTypes.UPDATE_DESKTOPS_FILTER_PAGE_SIZE_AND_CURRENT_PAGE),
+    mergeMap(() => [ new GetDesktops() ]));
```

Comment:
calling `GetDesktops()` as a side effect reload the items so that the proper amount show per page when the user has updated that selection.